src/server/adminInviteUser.ts

```diff
@@ -5,7 +5,12 @@ import { getSupabaseAdminAuthClient } from "@/supabaseAdminAuthClient";
 import { User } from "@supabase/gotrue-js";
 import { InviteUserDetails } from "@/app/admin/createUser/CreateUserForm";
 import supabase from "@/supabaseClient";
-import { logInfoReturnLogId } from "@/logger/logger";
+import { logErrorReturnLogId, logInfoReturnLogId } from "@/logger/logger";
+
+export type InviteUserErrorType =
+    | "Admin Authentication Failure"
```

Comment:
These string union types tend to use camelCase or just lower case. One reason is to signal that this isn't supposed to be used for a message for end-users. It's just for the code to identify which error has occurred. I gravitate towards camelCase though cause it's a little easier to refactor / rename with IDEs if we needed to.

```suggestion
    | "adminAuthenticationFailure" // (or "admin authentication failure")
```